cirq/protocols/apply_unitary.py

```diff
@@ -69,10 +70,13 @@ class ApplyUnitaryArgs:
             qubits that the gate is operating on).
     """
 
-    def __init__(self,
-                 target_tensor: np.ndarray,
-                 available_buffer: np.ndarray,
-                 axes: Iterable[int]):
+    def __init__(
+            self,
+            target_tensor: np.ndarray,
+            available_buffer: np.ndarray,
+            axes: Iterable[int],
+            *,  # Force keyword args
+            _is_subspace: bool = False):
```

Comment:
It doesn't make sense for a constructor argument to be private.

Reply:
_is_subspace is no longer needed.